tests/test_factory_subclassing.py

```diff
@@ -27,3 +28,37 @@ def test_factory_raises_config_error_for_unsupported_model() -> None:
 
         class MyFactory(ModelFactory):
             __model__ = Null
+
+
+@pytest.mark.parametrize("is_base_factory", [True, False])
```

Comment:
please split this into two different test cases. This makes the test difficult to read and comprehend.

Reply:
Good point, I split it.